ci/ci_setup.sh

```diff
@@ -14,3 +14,7 @@ rm -rf spec/reports # no stale spec reports from previous executions
 
 # Setup the environment
 rake bootstrap # Bootstrap your logstash instance
+
+# Set up some general options for the rspec runner
+echo "--order rand" > .rspec
+echo "--format CI::Reporter::RSpec" >> .rspec
```

Comment:
As I said, this is only for our jenkins. If we coded in rake test:core we force everyone to use it.